pilz_control/include/pilz_control/pilz_joint_trajectory_controller_impl.h

```diff
@@ -106,7 +108,7 @@ handleHoldRequest(std_srvs::TriggerRequest&, std_srvs::TriggerResponse& response
   JointTrajectoryController::preemptActiveGoal();
   triggerMovementToHoldPosition();
 
-  ros::Duration(JointTrajectoryController::stop_trajectory_duration_).sleep();
+  pilz_utils::sleep(ros::Duration(JointTrajectoryController::stop_trajectory_duration_));
```

Comment:
You certainly have a reason why you change this line. So far, however, I'm missing the test which makes use of this change.

Reply:
I will push the changes on the test once they are completed.

Reply:
@HiddenBug ready for review

Reply:
An alternative to this change would be to replace the [time.cpp file](https://github.com/ros/roscpp_core/blob/kinetic-devel/rostime/src/time.cpp#L447) in the unit test compilation unit.